Makefile

```diff
@@ -112,10 +116,9 @@ format:
 
 .PHONY: ci-check-format
 ci-check-format: format
-	@if [[ -n `git status --porcelain --untracked-files=no` ]]; then \
-	 	>&2 echo "ERROR: the following files are not formatted correctly"; \
-	 	>&2 echo "'git status --porcelain' reported changes in those files after a 'make format' :"; \
-		>&2 git status --porcelain --untracked-files=no; \
+	@if [[ -n `git status --porcelain` ]]; then \
+	 	>&2 echo "ERROR: the following files are not formatted correctly:"; \
+		>&2 git status --porcelain; \
```

Comment:
The old message was a bit more verbose and maybe easier to understand why changing it ?